hugo/content/installation-oss.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Installation From NGINX Repository"
+draft: false
+description: "Learn how to install NGINX Agent from OSS Repository."
+weight: 300
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-1216"
+categories: ["configuration"]
+doctypes: ["task"]
+---
+
```

Comment:
We should probably include a pre-requisites section that mentions that NGINX OSS is installed and a supported operating system with a link to our supported OS's. Example can be found here https://docs.nginx.com/nginx/admin-guide/installing-nginx/installing-nginx-plus/#prerequisites